### PR TITLE
BridgeHubs: XCM ExportMessage benchmark - just Rococo for now

### PR DESCRIPTION
**Pull request by acatangiu**
Fixes https://github.com/paritytech/parity-bridges-common/issues/1943

Depends on https://github.com/paritytech/polkadot/pull/6923

### CODE REVIEW

File: parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/weights/xcm/pallet_xcm_benchmarks_generic.rs [hunk 1]
```diff
@@ -61,39 +61,39 @@ impl<T: frame_system::Config> WeightInfo<T> {
 	// Storage: ParachainSystem PendingUpwardMessages (r:1 w:1)
 	// Proof Skipped: ParachainSystem PendingUpwardMessages (max_values: Some(1), max_size: None, mode: Measured)
 	pub(crate) fn report_holding() -> Weight {
-		Weight::from_parts(32_598_000 as u64, 0)
+		Weight::from_ref_time(28_493_000 as u64)
```
[the remark this answers is not in the document]
Comment (acatangiu) [reply parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/weights/xcm/pallet_xcm_benchmarks_generic.rs:64]: yeah, we'll generate these using the bot so the right HW is used, once #2344 is merged - the current weight changes are just an example showing `export_message()` weight is being generated from benchmark

**Comment by acatangiu** [comment]
bot bench $ xcm bridge-hub-rococo bridge-hubs pallet_xcm_benchmarks::generic

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable
**Logs**: https://gitlab.parity.io/parity/mirrors/cumulus/-/jobs/2559230

**Comment by ggwpez** [comment]
bot bench $ xcm bridge-hub-kusama bridge-hubs pallet_xcm_benchmarks::fungibles  
bot bench $ xcm bridge-hub-rococo bridge-hubs pallet_xcm_benchmarks::fungibles  
bot bench $ xcm bridge-hub-polkadot bridge-hubs pallet_xcm_benchmarks::fungibles  
bot bench $ xcm bridge-hub-polkadot bridge-hubs pallet_xcm_benchmarks::generic

**Comment by ggwpez** [comment]
Sorry commented it in the wrong MR :facepalm:

**Comment by ggwpez** [comment]
bot clean

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable
**Logs**: https://gitlab.parity.io/parity/mirrors/cumulus/-/jobs/2584500

**Pull request closed without merging**